src/measure_theory/function/jacobian.lean

```diff
@@ -1245,24 +1245,37 @@ begin
   refl
 end
 
+lemma det_smul_aux (v : ℝ) : ((1 : ℝ →L[ℝ] ℝ).smul_right v).det = v :=
```

Comment:
Is there a better place for this lemma? If the file has already been ported, you can leave the lemma here, but then add a porting note saying exactly where we should move this lemma once the port is done.

Reply:
I've renamed it to something more descriptive, and added a porting note with where to move it.